egs2/fleurs/asr1/README.md

```diff
@@ -1,3 +1,48 @@
+# How to use Auxiliary CTC Objectives
```

Comment:
Having all of this here feels a bit odd to me. Could I create a `README` similar to that of https://github.com/espnet/espnet/tree/master/egs2/TEMPLATE/tts1 ?

It will be useful to add information on interctc and self condition etc there as well. I don't think there is existing documentation on that.

Reply:
Sounds like a very good plan to me! Another option is to add in the `docs`